src/components/SourcesTable/formatters.js

```diff
@@ -236,7 +236,7 @@ export const getStatusTooltipText = (status, appTypes, errors = {}) => ({
 />);
 
 export const getAllErrors = (
-    { availability_status, availability_status_error, applications = [], endpoint = { authentications: [] } }
+    { availability_status, availability_status_error, applications = [], endpoints = [] }
```

Comment:
It might be better to use the first endpoint directly
```JS
{ availability_status, availability_status_error, applications = [], endpoints: [endpoint] = [] }
```